modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -208,6 +210,11 @@ def lob
       'BVA'
     end
 
+    def store_business_lines
+      metadata['consumer_benefit_type'] = lob
```

Comment:
shouldn't this be `= benefit_type`? If not, let's explain in a comment why these two metadata keys have the same value.

Reply:
Related to this, in my ticket for the PACT Act boolean, I'm creating a "form_data" key to store all schema values that we can store (per our chat in standup yesterday). It's going to look like this:

`{ "form_data" => { "potential_pact_act" => true }, "another_pact_act_boolean" => true }`

If you did the same thing here, we could have something like:

```
{ 
    "form_data" => {
        "benefit_type" => "FID",
        "potential_pact_act" => true, 
    },
   "evidence_type" => "upload" # This is the field that Drew added for SCs
}
```

Reply:
If we like something different than that, I can change my implementation, but wanted to share that as a starting point.

Reply:
`benefit_type` and `potential_pact_act` are both going to live under the "attributes" section of the schema, so I think they should live in the same place under `metadata` at the very least.

Reply:
Just documenting here that we discussed this as a group and have decided to keep data that comes into our schema under a `form_data` key, but that the structure within the metadata should be purposeful and not be tied to the hierarchy of any individual schema.

E.g. `v1.veteran.address.street` and `v2.veteran.location.street_address` should both be mapped down to `form_data/veteran_address_street` within the metadata. The schema's paths are not relevant to metadata.

Copies of data between the `form_data` grouping and storing the same value elsewhere for ease of querying/lookup is also encouraged, such as for PACT-related data where one piece of data might come in from the schema and another from a downstream service we query for the data. We **should not** move the schema boolean from `form_data` to place it in a `pact` group, rather we should copy the key & value into the `pact` grouping.

Reply:
Right. I'd forgotten than NODs don't really have a benefit type. I'm leaning toward a harder stance in that `form_data` should only ever store things that came from a schema. To wit, we should also not add keys that don't directly come from a schema key. 

My hope is that by taking this harder stance, we can accurately determine (with reason) what keys will be stored in form_data metadata by looking over a version's schema, and then there will be no surprises from extra keys being present.

Since NOD's schema doesn't have a benefit type, but does result in a central_mail_business_line, we should not store anything in the form_data, and the central_mail_business_line should sit outside the form_data metadata key as it's already written for the other appeal types.

Reply:
Now just storing central_mail_business_line outside of the form_data key in metadata as asked